repeat.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
ahahaha accidental commit, but also, i think you should be able to approximate what you're doing here with `--runs_per_test=50`

Reply:
Oops lol